README.md

```diff
@@ -31,6 +37,33 @@ Click `Verify Application` to make sure your input is valid.
 
 To enable Azure Authentication, check `Azure Active Directory Matrix-based security`
 
+## Version 1.0.0 Migration Instruction
+
+In version 1.0.0, this plugin upgrades from Microsoft identity platform v1.0 to v2.0. Before you upgrade ad plugin to version 1.0.0, please read following items first.
+
+1. Make sure your Jenkins server has enabled HTTPS. Microsoft identity platform v2.0 forces using HTTPS schema in reply uri for its applications. Please update the reply uri for your AAD application. You also need to update the url setting in `jenkins.model.JenkinsLocationConfiguration.xml` file and restart your Jenkins instance.
```

Comment:
Can we give a link to an official website to show how to enable HTTPS?

Reply:
Have added instructions for solution template created ones.